msisensor2/0.1-deb/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM biocontainers/biocontainers:v1.2.0_cv2
+MAINTAINER biocontainers <biodocker@gmail.com>
+LABEL software="msisensor2" \ 
+    container="msisensor2" \ 
+    about.summary="MSIsensor2 is a novel algorithm based machine learning, featuring a large upgrade in the microsatellite instability (MSI) detection for tumor only sequencing data, including Cell-Free DNA (cfDNA), Formalin-Fixed Paraffin-Embedded(FFPE) and other sample types" \
+    about.home="https://github.com/niu-lab/msisensor2" \ 
+    software.version="0.1-deb" \ 
+    version="1" \ 
+    base_image="biocontainers:v1.2.0_cv2" \
+    about.copyright="2019-2022 Beifang Niu & Xinyin Han" \
+    about.license="SPDX:GPL-3.0-only" \ 
```

Comment:
Hum, they indeed say gpl, but  no source provided, not fond of injecting a software delivered as binary only with no ref to source code